experimental/kubernetes-v2/README.md

```diff
@@ -0,0 +1,84 @@
+# Spinnaker on Kubernetes
+
+> *NOTE:* This is not intended for production use, as both Redis and Cassandra
+> are backed by memory currently, and are so far only stable in single node
+> configurations. Both of these issues should be fixed soon, however.
+
+These steps are really written intended for someone deploying to Kubernetes
+using Spinnaker. If you want to run Spinnaker on Kubernetes but deploy to
+another platform, read [how to
+configure](http://www.spinnaker.io/docs/target-deployment-configuration) it
+first.
+
+## If You're Feeling Paranoid...
+
+This setup relies on a few Docker images that you can build yourself if you
+don't trust the ones I provide. 
+
+1. `gcr.io/google-samples/cassandra:v8` can be rebuilt from
+   [here](https://github.com/kubernetes/kubernetes/tree/master/examples/cassandra/image).
+
+2. `gcr.io/kubernetes-spinnaker/cassandra-keys:v2` can be rebuilt from
+   `./images/cassandra`.
+
+3. `gcr.io/kubernetes-spinnaker/redis-cluster:v2` can be rebuilt from
+   `./images/redis`.
+
+4. `quay.io/spinnaker/<project name>:latest` be rebuilt at head of
+   https://github.com/spinnaker/<project name>
+
+## Prerequisites
+
+Make sure you have a running Kubernetes cluster, which is explained in more
+detail [here](http://www.spinnaker.io/v1.0/docs/target-deployment-setup#section-kubernetes-cluster-setup).
+The key takeaway is having a kubeconfig file sitting in `~/.kube/config` that
+can authenticate with the cluster you want to deploy Spinnaker to. 
+Once that is all squared away, make sure that running `$ kubectl config
+current-context` refers to the cluster you want to have Spinnaker running in.
+
+Next, in the editor of your choice, open up `./config/clouddriver.yml`, and
+examine the `dockerRegistry` subsection. If you want to deploy Spinnaker
+images to your Kubernetes cluster, you'll to fill in the details of your quay
```

Comment:
Other than in the image reference above, this is the first time `quay` is mentioned. Reading from the top of this document to here, it's not entirely clear what I need a quay account for (at least not yet).